tools/test/travis-ci/scancode-evaluate.py

```diff
@@ -129,8 +137,8 @@ def license_check(scancode_output_path):
                 # Ignore files that cannot be decoded
                 # check the next file in the scancode output
                 continue
-
-            if not has_spdx_text_in_analysed_file(scanned_file_content):
+            # temporary workaround for files with Permissive Binary Licenses
+            if not has_spdx_text_in_analysed_file(scanned_file_content) or has_binary_license(scanned_file_content):
```

Comment:
```suggestion
            if (
                    not has_spdx_text_in_analysed_file(scanned_file_content)
                    or has_binary_license(scanned_file_content)
            ):
```
Am I reading this right? You are marking the file as an offender if it has a binary license?

Reply:
This has changed in my latest commit, agreed with @0xc0170 to have it check for PBLs in the function has_permissive_text_in_scancode_output.